lightning/src/chain/package.rs

```diff
@@ -224,10 +224,11 @@ impl HolderHTLCOutput {
 	}
 }
 
-impl_writeable!(HolderHTLCOutput, 0, {
-	preimage,
-	amount
-});
+impl_writeable_tlv_based!(HolderHTLCOutput, {
+	(0, amount, 0),
```

Comment:
I don't quite understand when a default value is needed.

Reply:
It isn't anymore :).